src/cr/cube/dimension.py

```diff
@@ -675,7 +685,7 @@ def shimmed_dimension_transforms_dict(self) -> Dict:
         - "alias": Subvariables also have an alias that identifies them. It is stored
           in `dimensions[i].type.elements[j].value.references.alias`.
         """
-        shim = copy.deepcopy(self._dimension_transforms_dict)
+        shim = self._dimension_transforms_dict
```

Comment:
Makes sense.
Any changes made to shim will directly affect `self._dimension_transforms_dict` and vice versa. 

This will definitely solve the problem.
But this could potentially lead to unintended side effects if modifications to shim were meant to be isolated but I guess this is not the case.

Am I right?

Reply:
Well the shim has been modified for sure but with an `append` in this case, all the original part are not touched, so I think we're safe, but of course for this aspect I'd like to have @gergness opinion